### ISSUE TITLE
PerformanceObserver.observe() breaks inside next/image on chrome 69

### ISSUE THREAD

**Issue by huzaifarif** [vercel/next.js#35200]
### Verify canary release

- [X] I verified that the issue exists in Next.js canary release

### Provide environment information

```
Operating System:
  Platform: darwin
  Arch: arm64
  Version: Darwin Kernel Version 21.1.0: Wed Oct 13 17:33:24 PDT 2021; root:xnu-8019.41.5~1/RELEASE_ARM64_T8101
Binaries:
  Node: 15.14.0
  npm: 7.7.6
  Yarn: 3.0.1
  pnpm: N/A
Relevant packages:
  next: 12.0.10
  react: 0.0.4
  react-dom: 0.0.4
```

### What browser are you using? (if relevant)

Chrome 69.0.3497.106

### Describe the Bug

TL,DR
`performanceObserver.observe()` throws an exception when not provided with the `entryTypes` option **in older browsers like Chrome 69**.

Description:
When using next/image, it internally attaches performance observer for LargestContentfulPaint events.
On older browsers like Chromium 69 this fails with the following error:
`TypeError: Failed to execute 'observe' on 'PerformanceObserver': required member entryTypes is undefined.`

While the current implementation follows the latest w3c specs which specifies that `performanceObserver.observe()` must be passed either `entryTypes` array or `type` and `buffered` as options, next/image usses the later here: https://github.com/vercel/next.js/blob/canary/packages/next/client/image.tsx#L519

But Chrome 69 enforces that `entryTypes` must be provided to `observe()`, hence it throws an exception.

![image](https://user-images.githubusercontent.com/9393546/157623138-3ced7113-c562-42fe-ba76-0816c397a085.png)

### Expected Behavior

`performanceObserver.observe()` should be wrapped inside a try/catch block

### To Reproduce

Use next/image in an older browser like Chrome 69

**Comment by neeraj3029** [comment]
I agree with this change. I was also able to reproduce this issue on browserstack Chrome 65
![Screenshot 2022-03-10 at 3 44 14 PM](https://user-images.githubusercontent.com/31539812/157649186-7b05c736-465a-4564-8274-673c7934c0b2.png)
